core/src/main/java/org/hibernate/ogm/dialect/query/spi/RecognizerBasedParameterMetadataBuilder.java

```diff
@@ -24,32 +23,19 @@
 
 	@Override
 	public ParameterMetadataImpl buildParameterMetadata(String nativeQuery) {
-		ParamLocationRecognizer recognizer = new ParamLocationRecognizer();
+		// At the moment we don't care because neo4j doesn't support positional paramters
+		// I'm not even sure it would make a difference from the Neo4j point of view.
+		// We can change this in a separate issue and check the right value using SessionFactoryImplementor
+		// or SessionFactoryOptions
```

Comment:
I'm a bit puzzled by this comment. We are in the core module and you are only talking about Neo4j?

As far as I can see, you decided against taking into account the ORM option in OGM. Why is that?

Reply:
I thought the comment was clear, but it seems it's not.

Currently, positional parameters query don't work or are not supported. I was thinking about Neo4j because is the only dialect at the the moment that would be affected by this change.

Reply:
This issue is more connected to test positional parameters and  think we can work on it when solving this issue: https://hibernate.atlassian.net/projects/OGM/issues/OGM-1407

Reply:
Hmm, OK. So if it's not supported at all, let's be more generic in the comment and clearly states that it's not supported (and why not add a pointer to the issue).

That being said, does it take a lot of effort to mimic the ORM behavior and get ready for when it's supported? I'm a bit worried we could forget to update that later.

(If it's a lot of efforts, I understand we don't do the extra mile now though but we should add a note in the positional parameters issue description)

Reply:
I prefer to work on it when we work on the positional parameters issue because we need to add new tests to check that everything works as expected. I've updated the message and the issue, I'm confident we won't forget about it.

Reply:
OK, cool, thanks for updating the issue, that's all I asked for :).